### DISCUSSION TITLE
Errors 'GnuTLS: error in pull function' & 'Unable to establish SSL connection'

### DISCUSSION THREAD

**Discussion by krayneakkuraten** [community/community#133952] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi all, When trying to install Proton VPN in Kali Linux, I encountered the following errors: 
```
Errors 'GnuTLS: error in pull function' &
'Unable to establish SSL connection' 

```
**Request code from terminal**
```
(kali㉿kali)-[~]
└─$ wget https://repo.protonvpn.com/debian/dists/stable/main/binary-all/protonvpn-stable-release_1.0.3-3_all.deb --no-check-certificate
--2024-07-29 17:01:59--  https://repo.protonvpn.com/debian/dists/stable/main/binary-all/protonvpn-stable-release_1.0.3-3_all.deb
Resolving repo.protonvpn.com (repo.protonvpn.com)... 188.114.98.229, 188.114.99.229, 2606:4700:20::ac43:4672, ...
Connecting to repo.protonvpn.com (repo.protonvpn.com)|188.114.98.229|:443... connected.
GnuTLS: Error in the pull function.
Unable to establish SSL connection.
```
Can anyone help fix it?

**upd:** I apologize to everyone for the inconvenience, but if anyone encounters a similar problem, it is solved by enabling VPN.  I first downloaded the first VPN I could find for my browser, then downloaded another VPN via terminal, ran it there, re-entered the command to download Proton and everything downloaded without any problems.

**Comment by metagamma** [comment]
Ensure that the ca-certificates package is installed and up to date.

``` 
sudo apt install ca-certificates
sudo update-ca-certificates
```

**Reply by krayneakkuraten** [reply]
```
ca-certificates is already the newest version (20240203).
Summary:
  Upgrading: 0, Installing: 0, Removing: 0, Not Upgrading: 0

```
Packages are updated, any other suggestions where to dig? Can it be related to the fact that ProtonVPN site is blocked in my country?

**Comment by krayneakkuraten** [comment, marked as answer]
I first downloaded the first VPN I could find for my browser, then downloaded another VPN via terminal, ran it there, re-entered the command to download Proton and everything downloaded without any problems.
